src/components/Login/Login.jsx

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from 'react';
+import '../../App.css';
+import Button from '../../common/Button/Button';
+import Input from '../../common/Input/Inpit';
+import { useNavigate } from 'react-router-dom';
+import Header from '../Header/Header';
+
+export function Login(props) {
+	const [emailUser, setEmailUser] = useState(null);
+	const [passUser, setPassUser] = useState(null);
+	let navigate = useNavigate();
+
+	function handleClick() {
+		//navigate(`/invoices/${newInvoice.id}`);
+		navigate(`/registration`);
+	}
+
+	function onChangeEmail(text) {
+		setEmailUser(text);
+	}
+
+	function onChangePassword(text) {
+		setPassUser(text);
+	}
+
+	async function onSubmit(event) {
+		event.preventDefault();
+
+		const newUser = {
+			password: passUser,
+			email: emailUser,
+		};
+
+		const response = await fetch('http://localhost:4000/login', {
+			method: 'POST',
+			body: JSON.stringify(newUser),
+			headers: {
+				'Content-Type': 'application/json',
+			},
+		});
+
+		const result = await response.json();
+
+		if (result.successful === true) {
+			localStorage.setItem('courseUserToken', result.result);
+			navigate(`/courses`);
+			props.userName(result.user.name);
+		} else {
+			alert('The username or password is incorrect');
+		}
+	}
+
+	return (
+		<div>
+			<Header isRegistration={true} />
+			<div className='FormMain'>
+				<form onSubmit={onSubmit} style={{ width: '350px' }}>
+					<p style={{ textAlign: 'center', fontSize: '22px' }}>
+						<label>
+							<b>Login</b>
+						</label>
+					</p>
+					<Input
+						type='email'
+						name='Email'
+						placeholderText='Enter email'
+						onChange={onChangeEmail}
+					/>
+					<Input
+						type='password'
+						name='Password'
+						placeholderText='Enter password.'
+						onChange={onChangePassword}
+					/>
+
+					<p style={{ textAlign: 'center' }}>
+						<Button buttonText='Login' type='submit' />
+					</p>
+					<p style={{ textAlign: 'center' }}>
+						<label>
+							If you not have an account you can{' '}
+							<b>
+								<label onClick={handleClick}>
```

Comment:
you should always use `<Link` from react-router-dom for navigation for accessibility and usability reasons

Reply:
for example: you cannot right-click this to open in a new tab